src/System.ComponentModel.Composition/src/Microsoft/Internal/GenerationServices.cs

```diff
@@ -225,9 +223,8 @@ public static void AddLocalToLocalDictionary(this ILGenerator ilGenerator, Local
 
         /// Generates the code to get the type of an object and store it in a local
         /// <param name="ilGenerator"></param>
-        /// <param name="dictionary"></param>
-        /// <param name="key"></param>
-        /// <param name="value"></param>
+        /// <param name="exception"></param>
+        /// <param name="dataStore"></param>
```

Comment:
📝 This is why I leave 1573 disabled. This is a duplication of content two lines below.